quickstarts/gradle/vertx/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-bin.zip
```

Comment:
We should keep the wrapper versions aligned for all quick starts
so far we have, `7.2`, `7.1.1`, and `7.0`.

Can we switch this one to 7.2?